common/backoff.go

```diff
@@ -0,0 +1,29 @@
+package common
+
+import (
+	"time"
+	"github.com/cenkalti/backoff"
+)
+
+// Default values for PegExponentialBackOff.
+const (
+	DefaultInitialInterval     = 500 * time.Millisecond
+	DefaultRandomizationFactor = 0.5
+	DefaultMultiplier          = 1.5
+	DefaultMaxInterval         = 60 * time.Second
+	DefaultMaxElapsedTime      = 10 * time.Second  // max 10 seconds
+)
+
+// PegExponentialBackOff creates an instance of ExponentialBackOff
+func PegExponentialBackOff() *backoff.ExponentialBackOff {
+	b := &backoff.ExponentialBackOff{
+		InitialInterval:     DefaultInitialInterval,
+		RandomizationFactor: DefaultRandomizationFactor,
+		Multiplier:          DefaultMultiplier,
+		MaxInterval:         DefaultMaxInterval,
+		MaxElapsedTime:      DefaultMaxElapsedTime,
+		Clock:               backoff.SystemClock,
+	}
+	b.Reset()
```

Comment:
is Reset() really necessary on a just instantiated struct?

Reply:
That is a good point, I am not sure, kinda followed what was done here 
https://github.com/cenkalti/backoff/blob/v2.1.1/exponential.go#L83

Reply:
Ah ok, yeah sure let's keep it then.